### DISCUSSION TITLE
记本封面和背景设置无法添加图片

### DISCUSSION THREAD

**Discussion by Nebula0033** [WatsonYao/nian_app#78] [Q&A] [answered]
老师我想问一下，去年封面和背景图片都是正常添加的，相册里也是备份两份图片没有删过，也没动过文件夹文件，后来隔了一段时间没用，最近用的时候发现原先有封面的记本都变空白了，包括记本背景（但是头像图片还能正常显示）后来不管是编辑旧记本还是新建新记本，点击添加封面和背景都没有反应（像附带截图一样的，不知道您能不能看见，就是点击没有反应_(:з」∠)_）切换图片进展后也无法添加图片了，到设置里看了存储权限也打开了，不知道是哪里出问题了👉👈或者说清理手机内存的时候有可能会删除一些文件吗？如果是这样那该怎么恢复啊，更新了好像也没什么改变🤔
![Screenshot_20240807_010844](https://github.com/user-attachments/assets/f3baf349-196e-4b59-b3bc-5a4f266e0ce3)

**Comment by WatsonYao** [comment, marked as answer]
可以查看帮助文档相关问题的解释。

**Reply by Nebula0033** [reply]
帮助文档全都看完了没有改善才来问的。，一开始以为是资源文件没有了，在下载网址下载了resource.zip解压之后发现只有一堆系统自带图片，问题也没有解决😭

**Reply by WatsonYao** [reply]
![image](https://github.com/user-attachments/assets/3655f169-32a2-4271-9298-d3887498d09b)

**Reply by Nebula0033** [reply]
老师我不是图片显示不出来，我是没法添加图片👉👈

**Reply by WatsonYao** [reply]
首先，添加的图片要在手机内存里面，
其次，图片需要可访问权限，
最后，使用文件管理器添加图片，不要用系统自带的图片管理器，它可能没有图片的剪切等权限，只能查看和复制之类的。

**Reply by WatsonYao** [reply]
你说的看不太明白，不清楚是说以前的图片找不到了，还是现在添加不上了。以前的找不到了可以看新手指南和帮助文档，添加不上如上所述，你可以找个其他手机参考下，或者提供下你的手机信息，nian-个人资料那个页面截图发出来也可以。

**Reply by WatsonYao** [reply]
另外，不要使用应用双开，工作空间之类的，nian是不支持的。

**Reply by Nebula0033** [reply]
谢谢老师解答！非常感谢您这么耐心的解决我的问题，占用了您的时间真的非常抱歉！我不太擅长弄这种东西😣
我的问题是现在添加不上了，然后关于您说的办法，因为我不太清楚怎么使用文件管理器添加图片，上网搜索了也不是很确定做的对不对，总之尝试了在文件管理器中将图片复制粘贴到nian文件夹中的图片文件夹里，不过应用图库中还是没有这个图片（而且也没用应用双开（）
我把个人资料截图了，最后还是非常抱歉麻烦您了☕︎
![Uploading Screenshot_20240930_185138.jpg…]()

**Reply by Nebula0033** [reply]
抱歉老师我这边上传截图了但是刷新之后我没看见回复的图片，以防万一我把手机参数复制一下
设备参数
vivo-V1813BT
sdm710-qcom
sdk_int:28
x：1080 y:2207
xdp:360.0 ydp:735.6667 dpi:480
xdpi：318.976 ydpi：319.955
density:3.0 scaledDensity:3.35994

**Comment by WatsonYao** [comment]
你还是没有看懂新手指南。图片手动放nian文件夹肯定不会被识别成nian的图片。

**Comment by WatsonYao** [comment]
还是不要折腾图片了，只记录文本，或者使用支持图片上传其服务器的其他类似应用吧。如果无法理解或者规避系统的限制，是没办法描述或者解决这类问题的。我到现在都不知道你是想恢复之前不显示的图片，还是打算给记本重新换个封面或者背景。

**Reply by Nebula0033** [reply]
好的好的谢谢老师，麻烦您了₍˄·͈༝·͈˄*₎◞ ̑̑
